app/models/public_body/calculated_home_page.rb

```diff
@@ -0,0 +1,52 @@
+# Guess the home page based on the request email domain.
+module PublicBody::CalculatedHomePage
+  extend ActiveSupport::Concern
+
+  included do
+    cattr_accessor :excluded_calculated_home_page_domains, default: %w[
+      aol.com
+      btinternet.com
+      gmail.com
+      gmx.com
+      hotmail.com
+      icloud.com
+      mail.com
+      mail.ru
+      outlook.com
+      protonmail.com
+      qq.com
+      yahoo.com
```

Comment:
```suggestion
      yahoo.com
      yahoo.co.uk
```

We have `294` bodies with yahoo.co.uk. Another favourite for small bodies.